package.json

```diff
@@ -7,8 +7,15 @@
     "Editor",
     "WYSIWYG"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "ckeditor5-core": "ckeditor/ckeditor5-core"
```

Comment:
Whoops. I actually checked the manual, but haven't seen this :D Too many options!